cloud-common/src/main/java/com/sequenceiq/cloudbreak/util/BenchMark.java

```diff
@@ -0,0 +1,26 @@
+package com.sequenceiq.cloudbreak.util;
+
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import org.slf4j.Logger;
+
+public class BenchMark {
+    private BenchMark() {
+    }
+
+    public static void b(Runnable callback, Logger logger, String message, Object... params) {
```

Comment:
I love this method name. 😃

Reply:
It is statically importable (what a nice expression) so adding benchmark is is a simple `b(() ->` on the other side.

Reply:
But now the code is full these b()s, and it looks kinda weird. Naming it `measure` makes sense and not too long. Easy to understand (unlike static imported b()s)

Reply:
As i know `b` is a common name of benchmarking function. I saw it in many framework.

Reply:
But it is used in frameworks. On the other hand this is the production code of cloudbreak. I think there is a difference.

Reply:
I mean framework's public interface.

Reply:
I also think having a longer name might help to understand the intention of the method.